src/feature/Votes/Votes.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react';
+import * as Styled from './styles';
+import { useRequest } from '../../hooks/use-request';
+import { Review } from '../../types/types';
+
+type Props = {
+  reviewId: string | undefined;
+  votes: number | undefined;
+};
+
+type ResponseT = {
+  review: Array<Review>;
+};
+
+const Votes = ({ reviewId, votes }: Props) => {
+  const [addedVotes, setAddedVotes] = useState(0);
+  const { sendRequest, isError, errorMsg } = useRequest();
+
+  const handleOnDownClick = () => {
+    const res = (data: ResponseT) => {
+      setAddedVotes(data.review[0].votes);
+    };
+
+    if (votes) {
+      sendRequest('PATCH', `/reviews/${reviewId}`, res, {
+        inc_votes: -1,
+      });
+    }
+  };
+
+  const handleOnUpClick = () => {
+    const res = (data: ResponseT) => {
+      setAddedVotes(data.review[0].votes);
+    };
+
+    sendRequest('PATCH', `/reviews/${reviewId}`, res, { inc_votes: 1 });
+  };
```

Comment:
You need to handle the upvoting optimistically. 

At the moment, on a slow connection, its a really sluggish experience. You click the button and nothing happened for a bit, then it goes up. 
I could as an end user, think it wasnt working and keep clicking it - resulting in multiple requests happening